src/main/java/org/folio/rest/impl/SamlAPI.java

```diff
@@ -173,8 +173,8 @@ public void postSamlCallback(RoutingContext routingContext, Map<String, String>
     final URI originalUrl = relayStateUrl;
     final URI stripesBaseUrl = UrlUtil.parseBaseUrl(originalUrl);
 
-    Cookie csrfTokenCookie = routingContext.getCookie(CSRF_TOKEN);
-    if(csrfTokenCookie == null || !originalUrl.getQuery().contains(CSRF_TOKEN + "=" +csrfTokenCookie.getValue())) {
+    Cookie relayStateCookie = routingContext.getCookie(RELAY_STATE);
+    if(relayStateCookie == null || !relayState.contentEquals(relayStateCookie.getValue())) {
```

Comment:
"if (" ?  (a blank between if and ()

Reply:
> "if (" ? (a blank between if and ()

Done. Got that from the original PR. :).